packages/core/lib/send-result/node.ts

```diff
@@ -0,0 +1,93 @@
+import type { ServerResponse } from "http";
+import type { Http2ServerResponse } from "http2";
+import { HttpError } from "../errors";
+import type {
+  Response,
+  MultipartResponse,
+  Push,
+  ProcessRequestResult,
+} from "../types";
+
+type RawResponse = ServerResponse | Http2ServerResponse;
+
+export async function sendResponseResult(
+  responseResult: Response<any, any>,
+  rawResponse: RawResponse
+) {
+  rawResponse.statusCode = responseResult.status;
+  rawResponse.end(JSON.stringify(responseResult.payload));
+}
+
+export async function sendMultipartResponseResult(
+  multipartResult: MultipartResponse<any, any>,
+  rawResponse: RawResponse
+) {
+  rawResponse.writeHead(200, {
+    // prettier-ignore
+    "Connection": "keep-alive",
```

Comment:
Are there any customizations we need here? do people need to customize the headers sometimes?